docs/topics/api/auth_internal.rst

```diff
@@ -35,35 +35,15 @@ accessible through the VPN and requires using the following endpoints:
 
 The token is available in two forms:
 
-    * For the endpoints returning JSON, as a property called ``token``.
-    * For all endpoints, as a cookie called ``jwt_api_auth_token``. This cookie
+    * For the endpoints mentionned above, as a property called ``token``.
+    * For all endpoints, as a cookie called ``api_auth_token``. This cookie
       expires after 30 days and is set as ``HttpOnly``.
 
 
-Verifying a JWT
-===============
-
-You can verify that a token is valid by calling:
-
-.. http:post:: /api/v3/frontend-token/verify/
-
-    :<json string token: The JWT you want to verify.
-    :status 200: The token is valid.
-    :status 400: The token is invalid.
-
-If a 400 Bad Request error is returned, the body of the response may contain
-additional information explaining why the token is invalid.
-
-
 Creating an Authorization header
 ================================
 
-When making an authenticated API request, put your generated
-`JSON Web Token (JWT)`_ into an HTTP Authorization header prefixed with
-``Bearer``, like this::
+When making an authenticated API request, put your generated API Token into an
+HTTP Authorization header prefixed with ``Bearer``, like this::
 
     Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpYXQiOjE0NDcyNzMwOTZ9.MG9LJiEK5_Db8WpF5cWWRebXCtUB48EJzxKIBqQhSOo
```

Comment:
https://github.com/mozilla/addons-server/pull/4960/commits/8da7a16d12afa8b93438aae17c869a47054aef60 (but I might change it again if we change `secret` to something else)